src/kOS/Suffixed/Part/PartValue.cs

```diff
@@ -13,7 +15,7 @@ namespace kOS.Suffixed.Part
 {
     public class PartValue : Structure, IKOSTargetable
     {
-        private readonly SharedObjects shared;
+        protected readonly SharedObjects shared;
```

Comment:
Access from an inherited class is a little bit of a hybrid issue.  I agree in concept with not publicly exposing the member variable itself, and I don't think we want this variable publicly exposed anyways.  Protected fields are only accessible to inherited classes, which makes it helpful to not hold a 2nd handle to the object.  I think I've always treated protected fields similarly to private.  Would you prefer that I implement a protected property, or just add the 2nd handle to the DockingPortValue class?